code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -12,6 +12,7 @@
 	var/obj/item/clothing/suit/space/hardsuit/suit
 	item_color = "engineering" //Determines used sprites: hardsuit[on]-[color] and hardsuit[on]-[color]2 (lying down sprite)
 	actions_types = list(/datum/action/item_action/toggle_helmet_light)
+	dynamic_hair_suffix = ""
```

Comment:
Shouldn't be needed, it's already inheriting this from /obj/item/clothing/head/helmet/space